module/item/mixins/costEditMixin.js

```diff
@@ -0,0 +1,22 @@
+export let costEditMixin = {
+    attachHtmlListeners(html, callback, ...args) {
+        $(html).find('.component-cost').on('change', () => {
```

Comment:
There are rumors that jQuery will not be bundled with Foundry after v16 anymore. Do you want to tackle it already?

Reply:
Done 👍